test/metabase/api/dashboard_test.clj

```diff
@@ -165,7 +165,7 @@
                     dashboard-defaults
```

Comment:
we'll have tests to get values for parameter with `source_type=card` right?

Reply:
we already do

Reply:
just wrote them